spec/motion/configuration_spec.rb

```diff
@@ -5,16 +5,12 @@
     subject(:default) { described_class.new }
 
     before(:each) do
-      expect_any_instance_of(described_class).to(
-        receive(:warn).with(/Motion is automatically inferring/)
-      )
-
-      expect_any_instance_of(described_class).to(
-        receive(:`).with("git rev-parse HEAD").and_return(revision_from_git)
+      expect_any_instance_of(Motion::RevisionCalculator).to(
+        receive(:perform).and_return(revision_hash)
```

Comment:
Now that this doesn't have a global side effect, I would move it into a unit test in the `#revision` section.